cli/types/cypress.d.ts

```diff
@@ -241,6 +241,11 @@ declare namespace Cypress {
      */
     spec: Spec
 
+    /**
+     * Currently executing test runnable instance.
+     */
+    currentTest: Mocha.Test
```

Comment:
Can we get more specific typings for the properties on the `currentTest`? All the props show as 'any'.

Reply:
works fine for me
![image](https://user-images.githubusercontent.com/14625260/119398418-83b16980-bca5-11eb-8f6c-7abbfcb6677b.png)

I added type assertions on the typedef tests

Reply:
Should this be nullable? For CT at least, it can be.

Reply:
Eg: `currentTest: Mocha.Test | undefined`, or `currentTest?: Mocha.Test`?